### PR TITLE
Change context webui

### LINKED ISSUE #280: Provide way to change context (package / top-level) in Web UI
Use case:
IRIS BI editors - specifically the user portal - are misbehaving and not launching the web UI for some reason. (The page just reloads.)* As a workaround, I should be able to go back to the home page and get to the Git WebUI for my own package (or the top-l;evel repo for the namespace).

If the Web UI is launched without a context, it should help the user select one based on valid configuration: top level (if namespace temp folder has .git) and any git-configured packages. It should also be possible to change context after the fact.

*I don't know what the cause is, that's a separate issue (encountered in my development environment for a personal side project). $zv is IRIS for Windows (x86-64) 2022.3 (Build 606U) Mon Jan 30 2023 09:12:22 EST.

### PR DESCRIPTION
**Pull request by isc-hwojnick**
Fixes #280

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [2.4.1] - Unreleased
+## [2.5.0] - Unreleased
+
+### Added
+- New way to change context of Git-Source-Control from UI (#280)
```
Comment (isc-tleavitt) [inline CHANGELOG.md:11]: Let's say "Allow changing namespaces and IPM package context from web UI"

File: cls/SourceControl/Git/WebUIDriver.cls [hunk 2]
```diff
@@ -20,14 +20,16 @@ ClassMethod HandleRequest(pagePath As %String, InternalName As %String = "", Out
             set responseJSON = ..GetSettingsURL(%request)
         } elseif $extract(pagePath, 6, *) = "get-package-version"{
             set responseJSON = ..GetPackageVersion()
+        } elseif $extract(pagePath, 6, *) = "namespace"{
+            set responseJSON = $NAMESPACE
```
Comment (isc-tleavitt) [inline cls/SourceControl/Git/WebUIDriver.cls:24]: Should be actual JSON for consistency

Comment (isc-tleavitt) [reply cls/SourceControl/Git/WebUIDriver.cls:24]: Actually, this seems unused (we just use the URL in the client code) - should just remove.

**Review by isc-tleavitt — CHANGES_REQUESTED** [review]
A few small points, and please test as a user with just %Developer.

**Comment by isc-tleavitt** [comment]
@isc-hwojnick to confirm, did you test this as %Developer?

**Comment by isc-hwojnick** [comment]
I was waiting on the %Dev global fix before doing thorough %Develop testing

**Review by isc-tleavitt — APPROVED** [review]

**Pull request merged**